cspell.json

```diff
@@ -0,0 +1,81 @@
+{
+  "language": "en",
+  "words": [
+    "Blurhash",
```

Comment:
I had it in since we also import it from a module (which we can't rename), but it would be better to just allow these specific lines, rather than leaving this in, I guess. I will change take it out.